components/submit/AddURL.js

```diff
@@ -1,84 +0,0 @@
-// {
```

Comment:
This action is now performed within [components/submit/EditForm.js](https://github.com/ooni/test-lists-ui/blob/main/components/submit/EditForm.js)